GitUI/Editor/FileViewer.cs

```diff
@@ -1960,12 +1960,12 @@ protected override CommandStatus ExecuteCommand(int cmd)
                 case Command.FindNextOrOpenWithDifftool: internalFileViewer.InvokeAndForget(() => internalFileViewer.FindNextAsync(searchForwardOrOpenWithDifftool: true)); break;
                 case Command.FindPrevious: internalFileViewer.InvokeAndForget(() => internalFileViewer.FindNextAsync(searchForwardOrOpenWithDifftool: false)); break;
                 case Command.GoToLine: goToLineToolStripMenuItem.PerformClick(); break;
```

Comment:
Why not also
```suggestion
                case Command.GoToLine: return PerformClickIfAvailable(goToLineToolStripMenuItem);
```
?

Reply:
I hesitated between consistency (always wrap PerformClick) or simplicity (this feature is always available so the wrapping is not needed).
I still don't know what is the best. Maybe you're right, consistency...